packages/camera/camera_platform_interface/test/types/type_export_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Since this is really testing that the main interface file exports types, not the types themselves, these tests should be a new `group('exports' ...` in `test/camera_platform_interface_test.dart`, rather than a new test file here.

Reply:
Done.